CODE_OF_CONDUCT.md

```diff
@@ -93,6 +93,11 @@ The following behaviours from any participants within the nf-core community (inc
 - Use of sexual or discriminatory imagery, comments, jokes, or unwelcome sexual attention.
 - Verbal and text comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, age, religion, or work experience.
 
+Other more subtle behaviours are also against our code of conduct, and may result in an immediate ban:
+
+- Sending unsolicited direct-messages or emails to community members on Slack / other platforms
+- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core, and to promote alternatives
```

Comment:
```suggestion
- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core, and to promote alternatives.
```

Reply:
I think 'promoting alternatives' seems a bit strong to result in a ban... if someone goes 'oooh look at this new shiny new workflow manager' we all might be interested in looking into it..

I would be a bit clearer e.g.:

```suggestion
- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core for the purposes of promote alternatives
```

Reply:
```suggestion
- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core solely for the purposes of promoting alternatives
```